airflow/providers/amazon/aws/hooks/redshift_data.py

```diff
@@ -95,6 +100,12 @@ def execute_query(
 
         statement_id = resp["Id"]
 
+        if kwargs["ClusterIdentifier"] and kwargs["WorkgroupName"]:
+            raise ValueError("Specify either 'cluster_identifier' or 'workgroup_name', not both.")
+
+        if not kwargs["ClusterIdentifier"] and not kwargs["WorkgroupName"]:
```

Comment:
```suggestion
        if not cluster_identifier and not workgroup_name:
```

Reply:
Thank you, addressed in 56cc6d8ea13031ca610250735c8b0071065b233f